packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx

```diff
@@ -479,6 +479,7 @@ export const CommentTree = ({
                 </div>
               )}
               <CommentCard
+                key={`${comment.id}-${comment.text}`}
```

Comment:
I approved but do we need comment.text ? our comment IDs aren't enough? They should be unique right? The text will slow down rendering in theory.

Reply:
actually we want to re-render whenever comment.text changes, I added id in there but its actually not needed.